src/app/pages/system/bootenv/bootpool-attach/bootpool-attach-form.component.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef, Component,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import filesize from 'filesize';
+import { map } from 'rxjs/operators';
+import { JobState } from 'app/enums/job-state.enum';
+import { helptext_system_bootenv } from 'app/helptext/system/boot-env';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { DialogService, WebSocketService } from 'app/services';
+
+@UntilDestroy()
+@Component({
+  selector: 'bootpool-attach-form',
+  templateUrl: './bootpool-attach-form.component.html',
+  styleUrls: ['./bootpool-attach-form.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BootPoolAttachFormComponent {
+  isFormLoading = false;
+  route_success: string[] = ['system', 'boot'];
+  route_cancel: string[] = ['system', 'boot', 'status'];
```

Comment:
This was more of a pattern in EntityForm because `route_success` and `route_cancel` were configuration properties.
When these are used once, you can just put them in `router.navigate` directly.